### DISCUSSION TITLE
RS2 data processing

### DISCUSSION THREAD

**Discussion by 10go-L** [gmtsar/gmtsar#412] [Q&A] [answered]
Hello, I'm running GMTSAR to create RS2 interferograms. I'm doing multiple ones and I want to do a SBAS time-series at the end. I know there is no automatic way to do this using the current properties of GMTSAR so I'm doing individual interferograms. **My question is about the cropped area for the SLC. Do you know if there is any quick way to crop all the SLC to the same area?**
I haven't work on the issue yet but I know I will have to crop the SLC or the results. In the configuration file I can see there is a radar option to crop the SLC but no LAT LON option.
Do  you think it would be better to crop the unwrapped and coherence files at the end or better crop the SLC directly?
What would be the best way to do this?

Thanks,

OS: Ubuntu WSL2
GMTSAR version: 6.2.2

**Comment by Xiaohua-Eric-Xu** [comment, marked as answer]
I am no expert with RS2 data, but if your stack of data largely cover the same area, you could directly use `cut_slc` to do the cutting. Or, you could use the region_cut parameter and use `p2p_processing.csh` to cut things. There is currently no lat lon option, though you could quickly determine a position in lat lon coordinates with `SAT_llt2rat`. 

For very large stack, you should cut at the beginning. If your stack is not so large (e.g. 30), cutting at the end may give you more flexibility on the final area of product.

**Reply by 10go-L** [reply]
Thanks for your answer! I'll try to crop them at the beginning, then.
